R-package/tests/testthat/test_basic.R

```diff
@@ -569,6 +569,23 @@ test_that("lgb.cv() respects parameter aliases for objective", {
   expect_length(cv_bst$boosters, nfold)
 })
 
+test_that("lgb.cv() prefers objective in params to keyword argument", {
+  data("EuStockMarkets")
+  bst <- lgb.train(
```

Comment:
Should be `cv()` here.

Reply:
ah!! bad copy-pasting, thanks for catching that 😬 

updated in https://github.com/microsoft/LightGBM/pull/5007/commits/297a0a561c37f4facbcf7c77b0328c4fc096d40c